docker/Dockerfile_aarch64_910B

```diff
@@ -0,0 +1,171 @@
+FROM ubuntu:20.04 as export_image
+
+WORKDIR /tmp
+
+ARG http_proxy
+ARG https_proxy
+ARG PYVERSION=3.10.5
+ARG DEBIAN_FRONTEND=noninteractive
+ARG CHIP=all
+ARG ASCEND_BASE=/usr/local/Ascend
+ARG TOOLKIT_PKG=Ascend-cann-toolkit_*.run
+ARG KERNELS_PKG=Ascend-cann-kernels-*.run
+ARG TOOLKIT_PATH=$ASCEND_BASE/ascend-toolkit/latest
+ARG DEEPLINK_TAG_OR_COMMIT=6012186b03cff6eac6587e7a06dbaa590af6d5df
+ARG DEEPLINKEXT_TAG_OR_COMMIT=525678f2c4c227e1e8bf358259a19a578b67bc37
+ARG LMDEPLOY_TAG_OR_COMMIT=v0.5.3
+
+RUN sed -i 's@http://.*.ubuntu.com@http://mirrors.tuna.tsinghua.edu.cn@g' /etc/apt/sources.list && \
+    apt update && \
+    apt install --no-install-recommends ca-certificates -y && \
+    apt install --no-install-recommends bc wget -y && \
+    apt install --no-install-recommends curl gcc make g++ pkg-config unzip -y && \
+    apt install --no-install-recommends libsqlite3-dev libblas3 liblapack3 gfortran vim -y && \
+    apt install --no-install-recommends liblapack-dev libblas-dev libhdf5-dev libffi-dev -y && \
+    apt install --no-install-recommends libssl-dev zlib1g-dev xz-utils cython3 python3-h5py -y && \
+    apt install --no-install-recommends libopenblas-dev libgmpxx4ldbl liblzma-dev -y && \
+    apt install --no-install-recommends libicu66 libxml2 pciutils libgl1-mesa-glx libbz2-dev -y && \
+    apt install --no-install-recommends libreadline-dev libncurses5 libncurses5-dev libncursesw5 -y && \
+    apt install --no-install-recommends git gdb gcc-7 g++-7 -y && \
+    sed -i 's@http://mirrors.tuna.tsinghua.edu.cn@https://mirrors.tuna.tsinghua.edu.cn@g' /etc/apt/sources.list && \
+    apt clean && rm -rf /var/lib/apt/lists/*
+
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 7 \
```

Comment:
Why install both gcc-7 and gcc-9

Reply:
Is `gcov` necessary?

Reply:
OS embedded gcc is 9.4.0, but due to some strange compiler error, we must use gcc 7.5.0 for deeplink.framework. This update-alternatives command is a proper way to use gcc 7.5.0 default and remain gcc 9.4.0 available for OS.

> Why install both gcc-7 and gcc-9

Reply:
> Is `gcov` necessary?

I think gcov is not necessary, but the piece of code just makes gcc toolchain version to be same. update-alternatives only controls different versions of same programs. I refer update-alternatives manual [here](https://manpages.ubuntu.com/manpages/trusty/en/man8/update-alternatives.8.html).